[wasm] Stack overflow in `JIT/Directed/tailcall/tailcall/tailcall.sh` on Linux

Two tests failing with stack overflow on unrelated [PR](https://github.com/dotnet/runtime/pull/68696), [build](https://dev.azure.com/dnceng/public/_build/results?buildId=1777836&view=logs&jobId=190ad6c8-5950-568c-cadd-f2dfb7d5a79f&j=190ad6c8-5950-568c-cadd-f2dfb7d5a79f&t=c0f6fdc1-ac5d-583c-8ae1-a18de0846552), and [log](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-68696-merge-daba64bb137e43179e/JIT.Directed/1/console.76a77fac.log?helixlogtype=result):

```
  Starting:    JIT.Directed.XUnitWrapper (parallel test collections = on, max threads = 2)
    JIT/Directed/tailcall/tailcall/tailcall.sh [FAIL]
      
      Unhandled Exception:
      StackOverflowException
      [ERROR] FATAL UNHANDLED EXCEPTION: System.StackOverflowException: The requested operation caused a stack overflow.
         at Class1.Recurse4(Int32 depth, Object o1, Object o2)
         at Class1.Recurse3(Int32 depth)
         at Class1.Recurse2(Int32 depth, Object o1, Object o2)
         at Class1.Recurse1(Int32 depth)
         at Class1.Recurse4(Int32 depth, Object o1, Object o2)
         at Class1.Recurse3(Int32 depth)
         at Class1.Recurse2(Int32 depth, Object o1, Object o2)
...
```

And:

```
JIT/Directed/IL/Tailcall/ExplicitTailCallNoSO/ExplicitTailCallNoSO.sh [FAIL]
     
      Unhandled Exception:
      StackOverflowException
      [ERROR] FATAL UNHANDLED EXCEPTION: System.StackOverflowException: The requested operation caused a stack overflow.
         at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum)
         at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum)
         at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum)
         at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum)
         at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum)
         at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum)
         at ExplicitTailCallNoSO.CalculateSumWithTailCall(Int32 n, Int32 sum)
...
```

This was seen on other recent runs too.

--- a/src/tests/Directory.Build.props
+++ b/src/tests/Directory.Build.props
@@ -186,7 +186,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetOS)' == 'Browser'">
-    <CLRTestMSBuildArgs>/p:MSBuildEnableWorkloadResolver=false</CLRTestMSBuildArgs>
+    <CLRTestMSBuildArgs>/p:MSBuildEnableWorkloadResolver=false /p:Configuration=$(Configuration)</CLRTestMSBuildArgs>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(IsTestsCommonProject)' != 'true'">

--- a/src/tests/issues.targets
+++ b/src/tests/issues.targets
@@ -3285,13 +3285,6 @@
 
 
     <ItemGroup Condition=" '$(TargetArchitecture)' == 'wasm' " >
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/tailcall/**">
-            <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
-        </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/IL/Tailcall/ExplicitTailCallNoSO/*">
-            <Issue>https://github.com/dotnet/runtime/issues/69517</Issue>
-        </ExcludeList>
-
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Intrinsics/TypeIntrinsics_r/**">
             <Issue>https://github.com/dotnet/runtime/issues/54867</Issue>
         </ExcludeList>

